README.md

```diff
@@ -147,21 +163,42 @@ DID: `did:nft:{chainNamespace}.{chainReference}_erc1155.{contractAddress}_{token
 
 CAIP-29: `{chainNamespace}:{chainReference}/erc1155:{contractAddress}/{tokenId}`
 
-
 ### Conversions
+
 **DID->CAIP**
+
 ```
 const caip = did.substr(8).replace(/_/g, '/').replace(/\./g, ':')
 ```
+
 **CAIP->DID**
+
 ```
 const did = `did:nft:${caip.replace(/\//g, '_').replace(/:/g, '.')}`
 ```
 
+There are helpers that help you with the conversion:
+
+```typescript
+import { caipToDid, didToCaip } from 'nft-did-resolver'
+import { AssetId } from 'caip'
+
+// CAIP -> DID URL
+const assetId = new AssetId(
+  AssetId.parse('eip155:1/erc721:0x1234567891234567891234567891234596351156/0x1')
+)
+const didUrl = caipToDid(assetId) // did:nft:eip155.1_erc721.0x1234567891234567891234567891234596351156_0x1
+const didUrlWithTimestamp = caipToDid(assetId, 1628529680) // did:nft:eip155.1_erc721.0x1234567891234567891234567891234596351156_0x1?versionTime=2021-08-09T17:21:20Z
+
+// DID URL -> CAIP
+const assetId1 = didToCaip(didUrl) // eip155:1/erc721:0x1234567891234567891234567891234596351156/0x1
```

Comment:
Was using hex for asset_id standardized though caip yet?
What format are usually used on opensea and similar?

Reply:
Not standardised yet, but as stated in the spec, IMO, it makes sense to push for it.

Rarible and OpenSea represent token id as decimal string, which has to be converted to hex when using subgraph. Made the conversion via BigNumber (+18kb).

Reply:
FYI: https://github.com/ChainAgnostic/CASA/issues/5

Reply:
I'm kind of leaning the other way. Having the integer representation will just make the DID-URL easier for non devs to understand. We are already using BigNumber in our codebase afaik, and hopefully we can move to ES2020 soon enough.